test/strategy.test.ts

```diff
@@ -26,6 +26,9 @@ if (!strategy) throw 'Strategy not found';
 const example = require(`../src/strategies/${strategy}/examples.json`)[0];
 
 function callGetScores(example) {
+  example.addresses[0] = example.addresses[0].toLowerCase();
```

Comment:
Why lowerCase? 🤔

Reply:
In order to make sure strategies correctly handle address capitalization. See issue #790
The is already a test that checks to make sure a strategy returns same address capitalization or checksum, but it relies on the input address in each strategy's example data.
In our case, the example addresses were all lowercased, the strategy itself returned `toLowerCase()` addresses so the test passed. But in production metamask returned checksum addresses so there was a mismatch and all voting returned 0.
This makes sure input capitalization is "randomized" so that expected behavior is actually tested. This would have caught the issue.

Reply:
> But in production metamask returned checksum addresses so there was a mismatch and all voting returned 0.

Where is production you mean? 🤔 in score API? or snapshot.org, can share me the proposal where this happened?